Sources/KituraCORS/AllowedOrigins.swift

```diff
@@ -20,7 +20,11 @@ import Foundation
 
 /// Type alias for `AllowedOrigins.regex`.
 #if os(Linux)
+#if swift(>=3.1)
```

Comment:
Could you please fix the indentation in this block. Also why not just call it `RegexType` instead of `AllowedOriginsRegExType` ?